FilePersistence/src/version_control/Commit.cpp

```diff
@@ -107,7 +107,7 @@ bool Commit::isValidMatch(const char* content, qint64 size, const char* indexOfI
 								  bool findChildrenByParentId) const
 {
 	start = indexOfId-content;
-	end = start;
+	end = start+1;
```

Comment:
I do not understand how this change makes a difference. Could you explain with an example?